### DISCUSSION TITLE
Clipboard paste isn't working with renaming tab

### DISCUSSION THREAD

**Discussion by codezninja** [zellij-org/zellij#3832] [Q&A]
Not sure when this stopped working. But on the latest release at least. When I want to rename the tab and paste in what I have from my system clipboard. It doesn't paste into the tab but into the shell window instead. Anyone know how I can debug this?

**Comment by Lockszmith-GH** [comment]
Which OS, which terminal.

I'm on Windows, with either Tabby or Windows Terminal and Crtl+Shift+V pastes wherever I want

**Reply by codezninja** [reply]
MacOS 15.1.1 and Kitty 0.37.0
